### DISCUSSION TITLE
SONOFF TRVZB unsupported device

### DISCUSSION THREAD

**Discussion by chrRtg** [Koenkk/zigbee2mqtt#21776] [Devices]
Homeassistant with zigbee2mqtt edge.
Even after 5 retries to connect the device remain unsupported. It's a brand new SONOFF TRVZB.

In the device list the brand is shown as "not supported" but in the details it's discoverd as SONOFF.

Device Details:
Geräte-Typ
    EndDevice
Zigbee Modell
    TRVZB
Zigbee Hersteller
    SONOFF
IEEE Addresse
    0x38398ffffe1d96c2
Netzwerk Adresse
    0x86A1
Firmware-Datum
    20230811
Firmware-Version
    1.1.1

State File: 
[state.json.zip](https://github.com/Koenkk/zigbee2mqtt/files/14577549/state.json.zip)

Any suggestion on how to get the TRV integrated?

**Comment by Koenkk** [comment]
You are using an outdated z2m version probably, update to the latest edge by uninstalling/installing it.

**Reply by chrRtg** [reply]
Thanks mate! I've been runing a "edge" version, now switched to the main one with auto-update.
